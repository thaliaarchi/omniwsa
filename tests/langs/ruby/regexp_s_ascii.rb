puts "Ruby #{RUBY_VERSION}"
puts 'ASCII whitespace characters for Regexp \s:'
pattern = /^\s$/
(0..255).each do |c|
  s = c.chr(Encoding::ASCII_8BIT)
  if s =~ pattern
    puts "0x#{c.to_s(16).rjust(2, '0')} (#{s.inspect})"
  end
end
