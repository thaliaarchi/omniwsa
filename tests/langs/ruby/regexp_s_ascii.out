Ruby 3.3.6
ASCII whitespace characters for Regexp \s:
0x09 ("\t")
0x0a ("\n")
0x0b ("\v")
0x0c ("\f")
0x0d ("\r")
0x20 (" ")
